python/lsst/ts/atmcssimulator/enums.py

```diff
@@ -0,0 +1,50 @@
+# This file is part of ts_atmcssimulator.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from enum import Enum
+
+__all__ = ["Ack", "CommandKey"]
+
+
+class Ack(str, Enum):
+    ACK = "ack"
+    FAIL = "fail"
+    NOACK = "noack"
+    SUCCESS = "success"
```

Comment:
This is a duplication with `ts_atpneumaticssimulator`. Maybe in the future, you would consider to have a common repository for two AT CSCs.